test/spec/BpmnPropertiesPanelRenderer.spec.js

```diff
@@ -79,7 +80,8 @@ describe('<BpmnPropertiesPanelRenderer>', function() {
       moddleExtensions = {
         zeebe: ZeebeModdle
       },
-      description = {}
+      description = {},
+      layout = {}
```

Comment:
We use that to configure the opened properties panel groups before executing the a11y validation. With closed groups, there wouldn't be much to validate.

```js
layout: {
  groups: {
    'general': { open: true },
    'documentation': { open: true },
    'multiInstance': { open: true },
    'CamundaPlatform__Implementation': { open: true },
    'CamundaPlatform__AsynchronousContinuations': { open: true },
    'CamundaPlatform__JobExecution': { open: true },
    'CamundaPlatform__Input': { open: true },
    'CamundaPlatform__Output': { open: true },
    'CamundaPlatform__ConnectorInput': { open: true },
    'CamundaPlatform__ConnectorOutput': { open: true },
    'CamundaPlatform__ExecutionListener': { open: true },
    'CamundaPlatform__ExtensionProperties': { open: true },
    'CamundaPlatform__FieldInjection': { open: true }
  }
}
```

An alternative would be to open all groups manually (as done with the nested list), but that would lead to more boilerplate code I assume.